gk/main.c

```diff
@@ -1383,6 +1385,14 @@ process_pkts_front(uint16_t port_front, uint16_t port_back,
 			continue;
 		}
 
+		if (unlikely(packet.flow.proto == ETHER_TYPE_IPv4 &&
+						 !ipv4_configured_front ||
+						 packet.flow.proto == ETHER_TYPE_IPv6 &&
+						 !ipv6_configured_front)) {
```

Comment:
Style: the above three lines have too many tabs, please take a look at the [dpdk coding style](https://doc.dpdk.org/guides/contributing/coding_style.html#c-indentation). Note that, ``Line length is recommended to be not more than 80 characters, including comments.``